rmw_connext_cpp/src/rmw_client.cpp

```diff
@@ -22,10 +22,10 @@
 #include "rmw_connext_shared_cpp/types.hpp"
 #include "rmw_connext_shared_cpp/qos.hpp"
 
-#include "identifier.hpp"
+#include "rmw_connext_cpp/identifier.hpp"
 #include "process_topic_and_service_names.hpp"
 #include "type_support_common.hpp"
-#include "types/connext_static_client_info.hpp"
+#include "rmw_connext_cpp/connext_static_client_info.hpp"
```

Comment:
Agreed, I made the comment because your analog PR in rmw_fastrtps was complying and not this one. Apparently you fixed only this one though and not the other ones in this PR